dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeForm.java

```diff
@@ -0,0 +1,135 @@
+package com.dotcms.rest.api.v1.contenttype;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.contenttype.transform.contenttype.JsonContentTypeTransformer;
+import com.dotcms.repackage.com.fasterxml.jackson.core.JsonParser;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.DeserializationContext;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.JsonDeserializer;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.util.json.JSONArray;
+import com.dotmarketing.util.json.JSONException;
+import com.dotmarketing.util.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * {@link ContentTypeResource}'s form
+ */
+@JsonDeserialize(using = ContentTypeForm.ContentTypeFormDeserialize.class)
+public class ContentTypeForm  {
+
+    private final List<ContentTypeFormEntry> entries;
+    private final String requestJson;
+
+    public ContentTypeForm(final List<ContentTypeFormEntry> entries, final String requestJson) {
+        this.requestJson = requestJson;
+        this.entries = entries;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "entries". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)